app/styles/variables.css

```diff
@@ -126,6 +126,11 @@
   --button-success-border: #a7b6c233;
   --button-success-border-hover: #bfccd644;
   --button-success-box-shadow-focus: #2c974b44;
+
+  --color-gold-blur: rgba(255, 172, 47, 80%);
+  --color-silver-blur: rgba(192, 192, 192, 80%);
+  --color-leader-gold: rgb(233, 150, 6);
+  --color-co-leader-silver: #c0c0c0;
```

Comment:
Now that I think about it, it's better to move these variables to the `InterestGroupMemberList.css` file, since this is the only place where they are used. Variables in `variables.css` should be more "global" variables used throughout the app, to prevent it from becoming too crowded. The button variables above should likewise be moved, but you don't have to do that.